pyupgrade/_main.py

```diff
@@ -591,6 +592,19 @@ def visit_Call(self, node: ast.Call) -> None:
                     if not candidate:
                         i += 1
             else:
+                if self.min_version == (3, 6):
+                    nodes: List[ast.AST] = [node]
+                    while nodes:
+                        node_ = nodes.pop()
+                        if isinstance(node_, ast.Await):
+                            self.generic_visit(node)
+                            return
+                        for name in node_._fields:
+                            value = getattr(node_, name)
+                            if isinstance(value, ast.AST):
+                                nodes.append(value)
+                            elif isinstance(value, list):
+                                nodes.extend(value)
```

Comment:
might need to only do this for elements of `value` which as instances of `ast.AST`, I'll check later